lib/internal/bootstrap/node.js

```diff
@@ -183,7 +188,7 @@
     {
       // Install legacy getters on the `util` binding for typechecking.
       // TODO(addaleax): Turn into a full runtime deprecation.
-      const { pendingDeprecation } = process.binding('config');
+      const { pendingDeprecation } = internalBinding('options').getOptions();
```

Comment:
@burgerboydaddy I think something went wrong with this one?

Reply:
@addaleax initial request was to replace process.binding('config') to
internalBinding('options).getOptions(). Is there any reason why this line shouldn't be included?
If yes, I will restore this line to original one.
Please let me know what is your opinion.

thanks